tests/models/test_models_unet_2d_condition.py

```diff
@@ -673,7 +673,7 @@ def test_lora_save_torch_force_load_safetensors_error(self):
                 new_model.load_attn_procs(tmpdirname, use_safetensors=True)
             self.assertIn("Error no file named pytorch_lora_weights.safetensors", str(e.exception))
 
-    def test_lora_on_off(self):
+    def test_lora_on_off(self, expected_max_diff=1e-4):
```

Comment:
We should not need the `numpy_cosine_similarity_distance` in the model tests. Model test just test a single forward pass for which abs tensor comparison should be good enough. Let's maybe try with `1e-3` here - if this doesn't work, I'm happy to dive a bit deeper to see what's going on